src/lib/staged_ledger/check_commands.mli

```diff
@@ -1,20 +0,0 @@
-open Async
-open Mina_base
-module Ledger = Mina_ledger.Ledger
-
-type transaction_pool_proxy =
-  { find_by_hash :
-         Mina_transaction.Transaction_hash.t
-      -> Mina_transaction.Transaction_hash.User_command_with_valid_signature.t
```

Comment:
Oh, having `.mli` file was actually nice. Let's remove `module Ledger` line from it, but keep it